neon_audio/__init__.py

```diff
@@ -18,48 +18,5 @@
 # SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
 # WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
 # USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+from mycroft.audio.utils import wait_while_speaking, is_speaking, stop_speaking
```

Comment:
Just a note that signal checks can break here due to config; refactoring `is_speaking` out of a signal is on my short list though anyways

Reply:
the aim is to move this over completely to ovos_utils, long term plan

- ovos-core sets ovos_utils to read proper files
- ovos-core support yaml configs
- neon-core configs ovos-core
- ovos_utils uses the correct path all the time

since neon-core is creating the mycroft compatible config this should already be functional anyways no?

Reply:
I think that is correct; plan makes sense